beater/burrowbeat.go

```diff
@@ -86,21 +107,29 @@ func (bt *Burrowbeat) Stop() {
 func (bt *Burrowbeat) getConsumerGroupStatus(burrow map[string]interface{}) {
 	status := burrow["status"].(map[string]interface{})
 	group := status["group"].(string)
-	total_partitions := int(status["partition_count"].(float64))
-        total_lag := int(status["totallag"].(float64))
-
-	event := common.MapStr {
-		"@timestamp":		common.Time(time.Now()),
-		"type":			"consumer_group",
-		"count":		1,
-		"cluster":		bt.cluster,
-		"group":		group,
-		"total_partitions":	total_partitions,
-		"total_lag":		total_lag,
-		"burrow_status":	status,
+	partitions := status["partitions"].([]interface{})
+	for i, _ := range partitions {
```

Comment:
Why use the index and discard the value here? The very next line you index into `partitions` with the index, just do `for _, partition := partitions {`